packages/sw-runtime-caching/build.js

```diff
@@ -13,41 +13,18 @@
  limitations under the License.
 */
 
-const rollupBabel = require('rollup-plugin-babel');
-const path = require('path');
-const {buildJSBundle} = require('../../build-utils');
 const pkg = require('./package.json');
+const rollupBabel = require('rollup-plugin-babel');
+const {buildJSBundle, generateBuildConfigs} = require('../../build-utils');
+
+const plugins = [rollupBabel({
```

Comment:
If it doesn't hurt to have the commonjs and noderesolve plugins, could we just collapse these modules to just use the same set of plugins in a shared way?